source/partials/_site_header_index.html.erb

```diff
@@ -19,7 +19,7 @@
   />
 
   <div class="index-hero__text">
-    <h1 class="index-hero__title">Meet Sharesight</h1>
+    <h1 class="index-hero__title">Become a better investor</h1>
```

Comment:
The A/B tested copy was capitalized…though the Vimaly story is incorrect and did not reflect that capitalization.  So, not your fault, but should be updated..

I've questioned this in Vimaly to absolutely confirm, but the screenshots are from the A/B test and I'd rather not implement something different than they tested.

```suggestion
    <h1 class="index-hero__title">Become a Better Investor</h1>
```

Reply:
Yeah makes sense - have just updated the text such that it is capitalized but will wait for Vikas' response